build.sh

```diff
@@ -395,14 +401,18 @@ case "$COMMAND" in
         ;;
 
     "start-object-server")
-        # kill any object servers that are still running
-        (pgrep -f realm-object-server || true) | while read pid; do
-            kill $pid
-        done
+        kill_object_server
         ./sync/start-object-server.command
         exit 0
         ;;
 
+    "reset-object-server-between-tests")
+        kill_object_server
+        # Leave the server files alone to avoid 'bad_server_ident' errors
```

Comment:
I think that's a different issue. This issue is with sessions from older tests communicating with the server but finding out their server-side Realm files have been deleted from under them; I believe that issue is for multiple sessions being opened for the same file.